lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -241,6 +241,23 @@ describe('modules/manager/npm/extract/index', () => {
       ).toBeArrayIncludingOnly(['https://registry.example.com']);
     });
 
+    it('reads registryUrls from .yarnrc', async () => {
+      fs.readLocalFile = jest.fn((fileName) => {
```

Comment:
we should refactor this tests to use normal jest module mocking instead of constantly overwriting those functions, which will have side effects on following tests.

Reply:
Good idea!

I changed the setup to be similar to what is done elsewhere, but for some reasons the mock implementations don't seem to be applied. Do you have an idea what could be wrong here?